### ISSUE TITLE
[v3] - Invalid version number.

### ISSUE THREAD

**Issue by Kio-td** [Cog-Creators/Red-DiscordBot#1742]
# Other bugs

#### What were you trying to do?
Load the bot

#### What were you expecting to happen?

Bot stays quiet, or shows a splash

#### What actually happened?

```[29/05/2018 20:59] ERROR events on_error 181: Exception in on_ready
Traceback (most recent call last):
  File "/root/.pyenv/versions/3.6.5/lib/python3.6/site-packages/discord/client.py", line 224, in _run_event
    yield from coro(*args, **kwargs)
  File "/root/.pyenv/versions/3.6.5/lib/python3.6/site-packages/redbot/core/events.py", line 125, in on_ready
    if StrictVersion(data["info"]["version"]) > StrictVersion(red_version):
  File "/root/.pyenv/versions/3.6.5/lib/python3.6/distutils/version.py", line 40, in __init__
    self.parse(vstring)
  File "/root/.pyenv/versions/3.6.5/lib/python3.6/distutils/version.py", line 137, in parse
    raise ValueError("invalid version number '%s'" % vstring)
ValueError: invalid version number '3.0.0b15.post2'
```

#### How can we reproduce this issue?

Unsure. I just loaded the bot on a fresh and clean server.

**Comment by Tobotimus** [comment]
We apologise for this issue, and we are working to solve it, however for now the best workaround would be to downgrade back to beta 15 with `pip install --force-reinstall red-discordbot==3.0.0b15`. **However**, please note that installing this version requires a C compiler to be installed on your system. 

If you would rather not have to install a C compiler, you can install beta 15 without fuzzy command search with `pip install -U --force-reinstall --no-cache-dir https://github.com/mikeshardmind/Red-DiscordBot/archive/no-fuzz.zip#egg=red-discordbot`.

**Comment by mikeshardmind** [comment]
Note that none of the above options (b15post2, downgrading to b15, or using the non-supported branch without fuzzysearch) will work using the launcher. The launcher attempts a version comparison to see if an update is available, and will continue not working until b16 hits pypi

**Comment by tekulvw** [comment]
Temporarily fixed with #1770
